### PR TITLE
Basic Todo list Created using Go

### PR DESCRIPTION
**Pull request by Meru45**
Following operations are added 
-list
-add
-complete
-del

### CODE REVIEW

File: .todos.json [hunk 1]
```diff
@@ -0,0 +1 @@
+[{"Task":"Complete the issue","Done":false,"CreatedAt":"2023-10-15T14:05:18.413365+05:30","CompletedAt":"0001-01-01T00:00:00Z"}]
```
Comment (sudo-rsingh) [inline .todos.json]: add a new line

File: cmd/makefile
Comment (sudo-rsingh) [inline cmd/makefile:1]: what is this is this necessary

File: todo
Comment (sudo-rsingh) [inline todo:1]: what is this, is it necessary for new users. if not then write a .gitignore file

**Review by sudo-rsingh — CHANGES_REQUESTED** [review]

File: cmd/makefile
Comment (Meru45) [reply cmd/makefile:1]: yes, and Makefiles is to manage the dependencies of the source files of the programs during the compilation and linking (build) phase, that is, to compile only the files that need to be compiled by looking at the dependencies on each other and the last modified dates of the source files while the programs are being compiled.

File: todo
Comment (Meru45) [reply todo:1]: It is necessary

**Review by sudo-rsingh — APPROVED** [review]
LGTM

**Pull request merged**